src/TabPane.tsx

```diff
@@ -150,7 +150,7 @@ const TabPane: BsPrefixRefForwardingComponent<'div', TabPaneProps> =
 
     const prefix = useBootstrapPrefix(bsPrefix, 'tab-pane');
 
-    if (!active && !Transition && unmountOnExit) return null;
+    if (!active && !Transition && (mountOnEnter || unmountOnExit)) return null;
```

Comment:
We'll need to continue rendering the tab pane when it's not active after it has been shown once, similar to the behavior in react-transition-group.  I think we can track that in a ref here?

Reply:
Should the pane just be `display: none` in that case?

Reply:
Yeah, the `.active` class controls the `display: block` for the current visible pane, and the rest are set to `display: none` by default.

Reply:
Reworked the logic. I'd appreciate extra eyes on it – so many booleans 😂

Reply:
Just tested this and having just `unmountOnExit` is not working.  I think we might be able to get away with adding the extra case of "not active and mountOnEnter and tab was never shown" to the original condition to return null

Thinking like this:

```
    const hasMountedRef = React.useRef(false);

    if (
      !active &&
      !Transition &&
      (unmountOnExit || (mountOnEnter && !hasMountedRef.current))
    ) {
      return null;
    }

    ...

    hasMountedRef.current = true;

    return ...
```

We'll definitely need some tests here since there's several booleans like you mentioned 😛